### DISCUSSION TITLE
Optifine seems to crash

### DISCUSSION THREAD

**Discussion by DaCav289** [Sinytra/ForgifiedFabricAPI#56] [Q&A] [answered]
Hey there!

super new to this mod and so far had no issue running it besides when I had optifine installed, I was using the "preview_OptiFine_1.20.1_HD_U_I6_pre6" version and it seems to freeze and crash the forge client while loading, is there any alternatives or just wait till it works?

**Comment by Su5eD** [comment, marked as answer]
Optifine is not supported, please use an alternative like Rubidium or Embeddium

**Reply by DaCav289** [reply]
sweet thank you! 

I've never used those before either so something new to learn!
